arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -806,25 +849,25 @@ public void tracePostExecution(MessageFrame frame, Operation.OperationResult ope
       case DUP -> {}
       case SWAP -> {}
       case LOG -> {}
-      case CREATE -> this.romLex.tracePostOpcode(frame);
+      case CREATE -> romLex.tracePostOpcode(frame);
       case CALL -> {}
       case HALT -> {}
       case INVALID -> {}
       default -> {}
     }
   }
 
-  private void handleCreate(Address target) {
-    this.transients.conflation().deploymentInfo().newDeploymentAt(target);
-  }
-
   public int getCfiByMetaData(
       final Address address, final int deploymentNumber, final boolean deploymentStatus) {
     return this.romLex()
         .getCodeFragmentIndexByMetadata(
             ContractMetadata.make(address, deploymentNumber, deploymentStatus));
   }
 
+  public int callDataContextNumber(final boolean shouldCopyTxCallData) {
+    return shouldCopyTxCallData ? this.stamp() : 0;
```

Comment:
is this boolean still needed ? I recall we finally create a callFrame for the tx callData, even if it's empty, n the case someone does a CALLDATACOPY, we need to have in the callFrame list (I fixed this in a past replayTest imho ...)

Reply:
We will not be copying zeros from an empty calldata array. It's done without reference to any context